src/ApiCore/TransportFactory.php

```diff
@@ -201,54 +131,25 @@ protected static function createSslChannelCredentials()
         return ChannelCredentials::createSsl();
     }
 
-    /**
-     * @param string|null $transport
-     * @return string
-     */
-    private static function handleTransport($transport)
+    private static function defaultTransport()
     {
-        if ($transport) {
-            return strtolower($transport);
```

Comment:
Why remove this? It just handles cases where users provide an upper case string of 'REST' or 'gRPC'.

Reply:
I think we want to encourage users to get the casing right - particularly because we now support a $transportConfig array that is keyed to the transport. What if they supply `$transportConfig = ['GRPC' => [...]]`. Should we support that?